### ISSUE TITLE
[$250] Profile - Display name field displays live mark down when entering text with mark down

### ISSUE THREAD

**Issue by IuliiaHerets** [Expensify/App#53336]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.69-1
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**Email or phone of affected tester (no customers):** applausetester+232803@applause.expensifail.com
**Issue reported by:** Applause Internal Team

## Action Performed:
1. Go to staging.new.expensify.com
2. Go to Account settings > Profile.
3. Click Display name.
4. Enter text with mark down in name field.

## Expected Result:
The name field should not display live mark down because mark down is not supported for display name.

## Actual Result:
The name field displays live mark down when entering text with mark down.

## Workaround:
Unknown

## Platforms:

- [x] Android: Standalone
- [x] Android: HybridApp
- [x] Android: mWeb Chrome
- [x] iOS: Standalone
- [x] iOS: HybridApp
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

https://github.com/user-attachments/assets/d14ef106-7616-40b8-827e-c27878b53f40
  
</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~021866565523965770181</li>
        <li>Upwork Job ID: 1866565523965770181</li>
        <li>Last Price Increase: 2024-12-10</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @mallenexpensify</details>

**Comment by alitoshmatov** [comment]
Straightforward fix, we can go with @Themoonalsofall 's [proposal](https://github.com/Expensify/App/issues/53336#issuecomment-2509167673)

C+ reviewed 🎀 👀 🎀

**Comment by alitoshmatov** [comment]
Yes based on [C+ doc](https://docs.google.com/document/d/1BvohU05MTaHnjOD_vwJv_aqDAirv-ChkyRnKCAvOVyQ/edit?tab=t.0#bookmark=id.by6zfdtawcqn) I think we should process payment since PR was created and already reviewed by me before finding out duplicate PR.

cc: @anmurali @yuwenmemon

**Comment by mallenexpensify** [comment]
@Themoonalsofall can you please accept the job and reply here once you have? 
https://www.upwork.com/jobs/~021899587207378176091

Apologies for the mix up here.

**Comment by alitoshmatov** [comment]
> @alitoshmatov @anmurali could you please take a look this one? Thanks!

@Themoonalsofall I did try to catch some eyes couple of times. Just saying so that you don't think I ignored the your messages 😅 

<img width="593" alt="Image" src="https://github.com/user-attachments/assets/fea69caa-b49a-438d-8ca6-d456f5eb5ebb" />

**Comment by mallenexpensify** [comment]
Contributor: @Themoonalsofall paid $250 via Upwork
Contributor+: @alitoshmatov due $250 via NewDot

Apologies again, this was an edge case and shouldn't happen again.  Thanks for posting the pings in #contributor-plus Ali, sorry I missed those.

**Comment by JmillsExpensify** [comment]
$250 approved for @alitoshmatov
